sources/src/cache-base.ts

```diff
@@ -239,6 +243,23 @@ export class GradleStateCache {
         return fs.readFileSync(absolutePath, 'utf8')
     }
 
+    /**
+     * When the GitHub environment ACTIONS_RUNNER_DEBUG is true, run Gradle with --info and --stacktrace.
+     * see https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging
+     */
+    configureInfoLogLevel(): void {
+        const infoProperties = `org.gradle.logging.level=info\norg.gradle.logging.stacktrace=all\n`
```

Comment:
I used \n here, there is also os.EOL but I have no experience with TypeScript/nodeJS.

Reply:
This is fine, I think. An alternative that is particularly useful for long multi-line strings is to [use backticks and include the literal newlines](https://blog.enterprisedna.co/how-to-create-a-multiline-string-in-typescript/).

Reply:
Sorry, I was talking about Windows new line CRLN, don't know if the nodeJS api handles that as well (or if the properties file supports mixed line endings).